lib/adaptors/serialport.js

```diff
@@ -52,13 +52,13 @@ util.inherits(Adaptor, EventEmitter);
  */
 Adaptor.prototype.open = function open(callback) {
   var self = this,
-      port = this.serialport = new serialport.SerialPort(this.conn, {});
+      port = this.serialport = new serialport.SerialPort(this.conn, {}, false);
 
   function emit(name) {
     return self.emit.bind(self, name);
   }
 
-  port.on("open", function(error) {
+  port.open(function(error) {
```

Comment:
You probably need to stub the `open` method like `port.open = stub();` inside the `beforeEach` code here https://github.com/orbotix/sphero.js/blob/master/spec/lib/adaptors/serialport.spec.js#L12-L16